test/e2e/autoscale_hpa_test.go

```diff
@@ -0,0 +1,205 @@
+// +build e2e
+// +build hpa
```

Comment:
We should add this to the prow test scripts, so we see the test in action, right?

Reply:
Yes do you have a pointer?

Reply:
https://github.com/knative/serving/blob/main/test/e2e-tests.sh is where the tests are called.